swarm/network/kademlia.go

```diff
@@ -677,20 +677,33 @@ func (k *Kademlia) saturation(n int) int {
 func (k *Kademlia) full(emptyBins []int) (full bool) {
```

Comment:
Please add an explanation of the algorithm used here.

Reply:
So, as we talked about in the call, please add to the comment the definition of a node being healthy when "as healthy as can be," that is that it can have 0 connected peers if no peers are known in the bin. That's what threw me off, at least.